packages/gatsby-cli/src/init-starter.js

```diff
@@ -228,6 +228,15 @@ type InitOptions = {
   rootPath?: string,
 }
 
+const endInstruction = Path => {
+  report.info(`
+Successfully Added Gatsby site !!
+start developing your site now by running the following instructions
+  $ cd ${Path}
+  $ ${getPackageManager()} run develop
```

Comment:
This might not always work (it depends if starter is using `develop` script or not)

Reply:
We could replace this with `gatsby develop` since we assume that the user has `gatsby-cli` globally installed.

Reply:
What do you think about `npx`. I guess it will fit good here

Reply:
`npx` is okay but since the rest of the docs says `gatsby develop`, might be nice to keep it consistent. 

Also, they're very likely to have `gatsby` globally installed since they just ran `gatsby new` 😅

Reply:
What about, checking if gatsby-cli is installed, if not, show npx gatsby-cli develop? 🤷‍♂

Reply:
That's a good idea too! Since the user could've ran `npx gatsby new` 😂

Reply:
npx comes with node 8 so we should be good to go, unsure if it's npx gatsby or npx gatsby-cli

Reply:
Probably `npx gatsby-cli` since npx takes package name

Reply:
Checking `gatsby-cli` in local Or in global? 
Cause I guess for global, it will need workaround as path wont be same for diff os. Right?

Reply:
Yeah, I think let's just set it to `gatsby develop` for now since we encourage folks to install `gatsby-cli` globally in all documentation. We can revisit this later if necessary.

Reply:
Yeah using gatsby seems good to me. 
I will try to create a better package manager locating method later if required

Reply:
I changed it to `gatsby` 😃